app/components/self-clear-cache.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="self-clear-cache-container">
+    <div class="last-request">
+        Last Request: {{@time}}
+    </div>
+    <div class="clear-cache-button">
+        <button class="button" type="submit" disableb={{false}}>Clear cache</button>
+    </div>
+    <div class="remaining-requests">
+        {{@totalTimes}} / 3 requests remaining for today
+    </div>
+</div>
```

Comment:
Please read about Ember Test Selectors [here](https://guides.emberjs.com/release/testing/#toc_ember-test-selectors)

And name those selectors according to the tests written by Sagar in PR #79

Reply:
Okay! I will make the changes.